app/Models/NewsPost.php

```diff
@@ -204,9 +204,23 @@ public function editUrl()
         return 'https://github.com/'.OsuWiki::USER.'/'.OsuWiki::REPOSITORY.'/tree/master/news/'.$this->filename();
     }
 
-    public function firstImage()
+    public function firstImage($absolute = false)
```

Comment:
as turned out this isn't needed for this PR but it'll be needed for opengraph stuff, so... ┐(°～° )┌